klippy/extras/axis_twist_compensation.py

```diff
@@ -26,13 +26,24 @@ def __init__(self, config):
         self.calibrate_start_x = config.getfloat('calibrate_start_x')
         self.calibrate_end_x = config.getfloat('calibrate_end_x')
         self.calibrate_y = config.getfloat('calibrate_y')
-        self.z_compensations = config.getlists('z_compensations',
+        self.zx_compensations = config.getlists('zx_compensations',
```

Comment:
Might be a matter of taste, but I'd factorize these 4 variables in a single type (a named tuple or a dataclass, I'm not sure what's preferred) since they represent the same thing between x and y, maybe `AxisCalibration`. Then you can pass it as a single argument to `_get_interpolated_z_compensation` instead of three.

Reply:
Maybe it is a little bit overkill since I am using it twice; it might result in more lines